hummingbot/connector/connector/uniswap_v3/uniswap_v3_connector.py

```diff
@@ -55,11 +58,28 @@ def __init__(self,
     def name(self) -> str:
         return "uniswap_v3"
 
-    def get_order_price_quantum(self, trading_pair: str, price: Decimal) -> Decimal:
-        return Decimal("1e-6")
-
-    def get_order_size_quantum(self, trading_pair: str, order_size: Decimal) -> Decimal:
-        return Decimal("1e-6")
+    async def initiate_pool(self) -> str:
```

Comment:
I think this kind of issue should be done in a general ticket that aims to create a gateway class so as not to have inconsistency in gateway connectors.

Reply:
Refactoring this will require QA testing on older Uniswap version as well which is beyond testing scope for this PR. And since we have a Gateway connector refactor ticket #3306, I agree with Victor here that we could do it there then. 
The logic on `initiate_pool` is slightly different on each of the AMM connectors (Uniswap, Balancer and Uniswap v3), apart from the urls, I don't see concrete reasons why it has to be that way though. 
@vic-en  Also why do we need to set `self._trading_pairs` from the response?

Reply:
`resp["pairs"]` has the right order for the trading pair. This was one of the issues we had in the 1st ite which causes both positions created by the bot to fall on one side of the current pool price. So `self._trading_pairs = resp["pairs"]` stores the correct order which strategies can compare with what the user has configured.